client/my-sites/themes/themes-selection.jsx

```diff
@@ -86,9 +87,14 @@ class ThemesSelection extends Component {
 	}
 
 	onScreenshotClick = ( theme, resultsRank ) => {
+		const themes = this.props.themes;
 		trackClick( 'theme', 'screenshot' );
 		if ( ! this.props.isThemeActive( theme ) ) {
-			this.recordSearchResultsClick( theme, resultsRank, 'screenshot_info' );
+			this.recordSearchResultsClick(
+				find( themes, t => t.id === theme ),
```

Comment:
This has recently been fixed by #19020 (`recordSearchResultsClick` now accepts a theme ID as argument, as it should).